story/ep1/pt/umi1_12.txt

```diff
@@ -31,7 +31,7 @@
 `"E pensar que a Shannon‐chan realmente morreu...`
 ` Eu ainda não consigo acreditar.`
 ` ......Coitadinha...`
-` ......Não poder ser capaz de vê‐la de novo parece uma mentira...`
+` ......Não poder conseguir vê‐la de novo parece uma mentira...`
```

Comment:
unnecessary and strange.

"Não ser capaz" sounds better.

Reply:
Tem razão, vou reverter